spec/bundler/stub_specification_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+if Bundler.rubygems.provides?(">= 2.1")
+  RSpec.describe Bundler::StubSpecification do
```

Comment:
this should be `RSpec.describe Bundler::StubSpecification, :rubygems => ">= 2.1" do`